### ISSUE TITLE
ans1crypto abort error on OSX Catalina zsh shell

### ISSUE THREAD

**Issue by TylerJamesMalloy** [NYUCCL/psiTurk#386]
Running pip install psiturk installs the asn1crypto==0.24.0, package which has an issue with OSX Catalina zsh shell and needs to be upgraded to asn1crypto 1.2.0. For compatibility with OSX the version should be updated to asn1crypto 1.2.0 and all compatibility issues fixed. 

After installing psiturk with pip all pip functionality will be broken on the zsh shell and you will need to manually remove the package from your site-packages folder and reinstall it which gives the following error: 

ERROR: psiturk 2.3.6 has requirement asn1crypto==0.24.0, but you'll have asn1crypto 1.2.0 which is incompatible.

The error and solution to the ans1crypto issue are detailed here: 
https://github.com/Homebrew/homebrew-core/issues/44996

This error made it so that I was unable to run the psiturk-setup-example which is now running, though I don't know if there are any other issues caused by switching to asn1crypto 1.2.0

**Comment by deargle** [comment]
I just switched the python2 branch requiremenets.txt file to be generated using `pipreqs` instead of `pip freeze`. This removed stale package dependencies such as `asn1crypto` from the requirements file. Can you install from python2 branch to confirm that it works for you?

```
git clone git@github.com:NYUCCL/psiTurk.git
cd psiTurk
git checkout python2
pip install -e .
```

**Comment by TylerJamesMalloy** [comment]
Currently working for me. asn1crypto is not installed when running pip install -e . on that branch, and the issue is no longer present.

**Comment by deargle** [comment]
fixed in v2.3.7
